test/javascripts/acceptance/never-track-assign-reason-test.js.es6

```diff
@@ -0,0 +1,70 @@
+import { test } from "qunit";
+import {
+  acceptance,
+  query,
+  updateCurrentUser,
+} from "discourse/tests/helpers/qunit-helpers";
+import { visit } from "@ember/test-helpers";
+import { cloneJSON } from "discourse-common/lib/object";
+import topicFixtures from "discourse/tests/fixtures/topic";
+
+acceptance(
+  "Discourse Assign | Never track topics assign reason",
+  function (needs) {
+    needs.user();
+    needs.settings({
+      assign_enabled: true,
+      assigns_user_url_path: "/",
+    });
+
+    needs.pretender((server, helper) => {
+      server.get("/t/44.json", () => {
+        let topic = cloneJSON(topicFixtures["/t/130.json"]);
```

Comment:
O nice :+1: I didn't know we have a `cloneJSON` helper function before this.

Reply:
Yes very handy for modifying fixtures, a helper for this combined with some sort of auto generation of fixture data + a contract would make fixtures way less work for us.